libs/vkd3d/device.c

```diff
@@ -4849,7 +4725,242 @@ static HRESULT STDMETHODCALLTYPE d3d12_device_CreateProtectedResourceSession1(d3
     return E_NOTIMPL;
 }
 
-CONST_VTBL struct ID3D12Device7Vtbl d3d12_device_vtbl =
+static D3D12_RESOURCE_ALLOCATION_INFO* STDMETHODCALLTYPE d3d12_device_GetResourceAllocationInfo2(d3d12_device_iface *iface,
```

Comment:
Odd that we're returning a pointer. Is this some COM ABI shenanigans?

Reply:
See above.